src/wallet/wallet.cpp

```diff
@@ -2734,6 +2735,15 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletT
             return false;
         }
     }
+
+    LogPrintf("Fee Calculation: Fee:%s Bytes:%u Tgt:%d (requested %d) Reason:\"%s\" Decay %.5f: Estimation: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
```

Comment:
Should this be `LogPrint(BCLog::ESTIMATEFEE...` (ie only log if estimatefee logs are enabled)?